documentation/assemblies/upgrading/assembly-upgrade-resources.adoc

```diff
@@ -51,15 +51,33 @@ metadata:
 The following procedures describe the steps to update specific resources to use the `{KafkaApiVersion}` API version:
 
 * xref:proc-upgrade-kafka-resources-{context}[]
+* xref:proc-upgrade-zookeeper-api-version-{context}[] 
+* xref:proc-upgrade-topic-operator-api-version-{context}[] 
+* xref:proc-upgrade-entity-operator-api-version-{context}[] 
+* xref:proc-upgrade-cruise-control-api-version-{context}[]
+* xref:proc-upgrade-kafka-api-version-{context}[] 
```

Comment:
Since these cover the same resource ... can we group them? I.e. do something like this:

```
* Update the Kafka custom resource:
    ** xref:proc-upgrade-kafka-resources-{context}[]
    ** xref:proc-upgrade-zookeeper-api-version-{context}[] 
    ** xref:proc-upgrade-topic-operator-api-version-{context}[] 
    ** xref:proc-upgrade-entity-operator-api-version-{context}[] 
    ** xref:proc-upgrade-cruise-control-api-version-{context}[]
    ** xref:proc-upgrade-kafka-api-version-{context}[] 
* xref:proc-upgrade-kafka-connect-resources-{context}[]
```

I think that would make it a bit more clear that these are separate procedures but doing the same.

Reply:
OK. I separated these into the Kafka procedures and the "other custom resources".